.github/workflows/s3Deploy.yml

```
@@ -0,0 +1,35 @@
+name: Deploy to S3
+
+on:
+  push:
+    branches:
+      - main
+
+# Required for authenticating with AWS IAM
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::693696263829:role/token-list-github-action-role
+          role-session-name: githubactionsession
+          aws-region: eu-central-1
+
+      - name: Upload to S3
+        run: |
+          aws s3 cp src/public/CowSwap.json s3://files.cow.fi/tokens/CowSwap.json
```

This doesn't match the execution example, but if it worked for the other file, should work for the list. 

Should we test it with once we have the "new release" trigger config:
```suggestion
          aws s3 cp src/public/CowSwap.json s3://files.cow.fi/tokens/CowSwap-test.json
```

Reason: if we override wrongly this file, it will affect production. So maybe it's a good idea, to make a test of everything using a different target file.

---

I was testing with a different file and it worked, but testing with the actual file makes more sense.